libs/wire-api/src/Wire/API/Asset.hs

```diff
@@ -396,10 +396,6 @@ instance S.ToParamSchema (AssetLocation r) where
       & S.type_ ?~ S.SwaggerString
       & S.format ?~ "url"
 
-instance AsHeaders '[AssetLocation r] Asset (Asset, AssetLocation r) where
-  toHeaders (asset, loc) = (I loc :* Nil, asset)
-  fromHeaders (I loc :* Nil, asset) = (asset, loc)
```

Comment:
What's the motivation for this removal that appears unrelated to this PR?

Reply:
In this PR a generic version of this is introduced so this instance is obsolete.